python/lsst/ts/atmcssimulator/mcs_simulator.py

```diff
@@ -0,0 +1,193 @@
+# This file is part of ts_atmcssimulator.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+import logging
+import types
+import typing
+
+import jsonschema
+from lsst.ts import tcpip
+
+from .enums import Ack, CommandKey
+from .mcs_server_simulator import McsServerSimulator
+from .schemas.registry import registry
+
+__all__ = ["McsSimulator"]
+
+CMD_ITEMS_TO_IGNORE = frozenset({CommandKey.ID, CommandKey.VALUE})
+
+
+class McsSimulator:
+    def __init__(self) -> None:
+        self.log = logging.getLogger(type(self).__name__)
+        self.cmd_evt_server = McsServerSimulator(
+            host=tcpip.LOCALHOST_IPV4,
+            port=5000,
+            log=self.log,
+            dispatch_callback=self.cmd_evt_dispatch_callback,
+            name="CmdEvtMcsServer",
+        )
+        self.telemetry_server = McsServerSimulator(
+            host=tcpip.LOCALHOST_IPV4,
+            port=6000,
```

Comment:
I think you could discuss the ports with Patricio. If I correctly understood Patricio's meanings in the previous time, the ATMCS and ATPneumatic control system (labview) are deployed in the same cRIO. Therefore, if I was not wrong, you might need to have four different port numbers.